CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -11,4 +11,158 @@ def get_ddb_connection(env:str=os.getenv('Environment', '')) -> 'botocore.client
     return ddbclient
 
 def layer_test() -> str:
-    return "hello world"
\ No newline at end of file
+    return "hello world"
+
+class Fridge:    
+    REQUIRED_FIELDS = ['display_name', 'fridge_state', 'address', 'lat', 'long']
+    TABLE_NAME = "fridge"
+    STATES ={'AK', 'AL', 'AR', 'AZ', 'CA', 'CO', 'CT', 'DC', 'DE', 'FL', 'GA',
+           'HI', 'IA', 'ID', 'IL', 'IN', 'KS', 'KY', 'LA', 'MA', 'MD', 'ME',
+           'MI', 'MN', 'MO', 'MS', 'MT', 'NC', 'ND', 'NE', 'NH', 'NJ', 'NM',
+           'NV', 'NY', 'OH', 'OK', 'OR', 'PA', 'RI', 'SC', 'SD', 'TN', 'TX',
+           'UT', 'VA', 'VT', 'WA', 'WI', 'WV', 'WY'}
+    FOOD_ACCEPTS = []#TODO: Fill this in
+    FOOD_RESTRICTIONS = []#TODO: fill this in
+    FRIDGE_TYPES = {
+        'display_name': 'S',
+        'username': 'S',
+        'fridge_state': 'S',
+        'address': 'S',
+        'instagram': 'S',
+        'info': 'S',
+        'url': 'S',
+        'neighborhood': 'S',
+        'organizer_email': 'S',
+        'tags': 'L',
+        'food_accepts': 'L',
+        'food_restrictions': 'L',
+        'lat': 'S',
+        'long': 'S',
+        'last_editted': 'N',
+        'profile_image': 'S',
+        'check_in_time': 'S',
+        'check_in_notes': 'S',
+        'check_in_status': 'S',
+        'check_in_image': 'S'
+    }
+
+    def __init__(self, db_client, fridge: dict={}):
```

Comment:
Empty dict as a default argument is a [dangerous](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python) idea. Let's set the default as None

Reply:
thank you! I chased it to None :)